go.mod

```diff
@@ -5,7 +5,6 @@ go 1.16
 require (
```

Comment:
what is the reason for the commit 40b7a0c24eff6c8a9acb65af181914428fcfec97 change?
should it come before the previous commit?

Reply:
Done.